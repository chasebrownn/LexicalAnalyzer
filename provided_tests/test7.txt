t8 ((a).(b))|(c) #
" abc cab "