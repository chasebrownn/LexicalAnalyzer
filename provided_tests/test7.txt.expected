t8 , "ab"
t8 , "c"
t8 , "c"
t8 , "ab"
